packages/devtools_app/lib/src/screens/network/network_service.dart

```diff
@@ -13,38 +13,55 @@ class NetworkService {
 
   final NetworkController networkController;
 
-  /// Updates the last refresh time to the current time.
+  /// Updates the last Socket data refresh time to the current time.
   ///
-  /// If [alreadyRecordingHttp] is true it's unclear when the last refresh time
-  /// would have occurred, so the refresh time is not updated. Otherwise,
-  /// [NetworkController.lastRefreshMicros] is updated to the current
-  /// timeline timestamp.
+  /// If [alreadyRecordingSocketData] is true, it's unclear when the last
+  /// refresh time would have occurred, so the refresh time is not updated.
+  /// Otherwise, [NetworkController.lastSocketDataRefreshMicros] is updated to
+  /// the current timeline timestamp.
   ///
-  /// Returns the current timestamp.
-  Future<int> updateLastRefreshTime({bool alreadyRecordingHttp = false}) async {
+  /// Returns the current timeline timestamp.
+  Future<int> updateLastSocketDataRefreshTime({
+    bool alreadyRecordingSocketData = false,
+  }) async {
     // Set the current timeline time as the time of the last refresh.
     final timestampObj =
         await serviceConnection.serviceManager.service!.getVMTimelineMicros();
 
     final timestamp = timestampObj.timestamp!;
-    if (!alreadyRecordingHttp) {
-      // Only include HTTP requests issued after the current time.
-      networkController.lastRefreshMicros = timestamp;
+    if (!alreadyRecordingSocketData) {
+      // Only include Socket requests issued after the current time.
+      networkController.lastSocketDataRefreshMicros = timestamp;
     }
     return timestamp;
   }
 
+  /// Updates the last HTTP data refresh time to the current time.
+  ///
+  /// If [alreadyRecordingHttp] is true it's unclear when the last refresh time
+  /// would have occurred, so the refresh time is not updated. Otherwise,
+  /// [NetworkController.lastHttpDataRefreshTime] is updated to the current
+  /// time.
+  Future<void> updateLastHttpDataRefreshTime({
```

Comment:
Does this need to be `async`? We're not doing any async work.

Reply:
Removed `async`.